lib/segment/src/common/operation_error.rs

```diff
@@ -69,6 +69,14 @@ impl OperationError {
             backtrace: Some(Backtrace::force_capture().to_string()),
         }
     }
+
+    /// Create a new service error with a description and no backtrace
+    pub fn service_error_light(description: impl Into<String>) -> OperationError {
```

Comment:
How about naming this `service_error` and the other `service_error_with_backtrace`? Or are there just too many.

Reply:
I thought about this but there are 155 calls, quite the noisy PR.

We could try to migrate some call sites to the light version over time.